designsafe/apps/api/views.py

```diff
@@ -155,3 +156,41 @@ def post(self, request):
             },
         )
         return HttpResponse("OK", status=202)
+    
+
+
+class SystemMonitorProxyApi(BaseApiView):
+    """"
+    Proxy API for fetching system queue data from Core Portal. 
+    """
+
+    def get(self, request, hostname):
+        CORE_PORTAL_API_BASE = "https://cep.tacc.utexas.edu/api/system-monitor"
+
+        url = f"{CORE_PORTAL_API_BASE}/{hostname}"
+        print(f"Proxy API calling: {url}") 
+
+        
+        try:
+            response = requests.get(url, timeout=10)
+            response.raise_for_status()
+            return JsonResponse(response.json(), safe=False)
```

Comment:
Instead of `safe=False`, we can return a dict here of the form `{response: response.json(), status: 200}`